widget/playground/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -87,6 +87,7 @@ export function MultiSelect(props: MuliSelectPropTypes) {
           ) : (
             <TokensPanel
               list={list}
+              configTokens={props.configTokens}
```

Comment:
Please check all usage of config tokens, it seems we have some codes that still need update.

![Screenshot from 2023-12-26 22-33-12](https://github.com/rango-exchange/rango-client/assets/99180934/70631f95-7920-4dd7-95f9-ebb3dae5c911)

Reply:
Please double check all usages like this.

![Screenshot from 2023-12-26 23-02-26](https://github.com/rango-exchange/rango-client/assets/99180934/41c2ed3f-5fc2-4afb-aa2a-2fbbe8e15ccc)

Reply:
fixed